Gems/EMotionFX/Code/EMotionFX/CommandSystem/Source/ImporterCommands.cpp

```diff
@@ -113,15 +103,15 @@ namespace CommandSystem
             GetCommandManager()->ExecuteCommandInsideCommand(AZStd::string::format("Select -actorID %i", actor->GetID()).c_str(), outResult);
         }
 
-
         // mark the workspace as dirty
         m_oldWorkspaceDirtyFlag = GetCommandManager()->GetWorkspaceDirtyFlag();
         GetCommandManager()->SetWorkspaceDirtyFlag(true);
 
         // return the id of the newly created actor
         AZStd::to_string(outResult, actor->GetID());
 
-        EMotionFX::GetActorManager().RegisterActor(AZStd::move(actor));
+        // Register actor asset.
+        EMotionFX::GetActorManager().RegisterActor(actorAsset);
```

Comment:
An `AZ::Data::Asset` has similar semantics to a `shared_ptr`, in that copying one will increment an atomic int, and destroying one will decrement an atomic int. Since this function isn't using the `actorAsset` after this call, this should move the asset when registering, to avoid those atomic int increment/decrement calls.
```suggestion
        // Register actor asset.
        EMotionFX::GetActorManager().RegisterActor(AZStd::move(actorAsset));
```